sys/pm_layered/Kconfig

```
@@ -5,7 +5,7 @@
 # directory for more details.
 #
 
-config MOD_PM_LAYERED
+config MODULE_PM_LAYERED
     bool "Platform-independent Power Management"
-    depends on MOD_PERIPH_PM
+    depends on MODULE_PERIPH_PM
```

Shouldn't this rather be:
```suggestion
    depends on USEMODULE_PERIPH_PM
```
?

---

No, both symbols (`MODULE_PM_LAYERED` and `MODULE_PERIPH_PM`) are part of the Kconfig module migration (that's why they depend on `TEST_KCONFIG` as well).